docs/source/ko/_toctree.yml

```
@@ -135,8 +135,8 @@
     title: 커뮤니티 리소스
   - local: troubleshooting
     title: 문제 해결
-  - local: in_translation
-    title: (번역중) Interoperability with GGUF files
+  - local: gguf
+    title: (번역중) GGUF 파일들과의 상호 운용성
```

```suggestion
    title: GGUF 파일들과의 상호 운용성
```
`(번역중)`은 번역 완료 이후에는 삭제해도 될 것 같습니다!